src/metabase/search/postgres/core.clj

```diff
@@ -88,17 +88,22 @@
        (map :legacy_input)
        (map #(json/parse-string % keyword))))
 
-(defn search-minimal
-  "Perform a basic search that only uses the index"
-  [search-ctx]
-  (minimal (:search-string search-ctx)
-           (dissoc search-ctx :search-string)))
+(def ^:private default-engine hybrid-multi)
+
+(defn- search-fn [search-engine]
+  (case search-engine
+    :hybrid       hybrid
+    :hubrid-multi hybrid-multi
```

Comment:
I am so typo prone 🤦 

Leaving this for another day as I don't want another round of flakes.